### DISCUSSION TITLE
Approach to eradicate residual AF488

### DISCUSSION THREAD

**Discussion by githubannimal** [IBEXImagingCommunity/ibex_imaging_knowledge_base#56] [Q&A] [answered]
We are having great results with the bleaching of AF555 and AF647 (!) but AF488 still persists at a low level. We are using a manual approach using 1mg/ml solution of LiBH4 and have extended the incubation times to 1 hr without much improvement on the AF488 bleaching.

Would you recommend that we try 1.5 or 2mg/ml solution?

**Comment by preshar** [comment]
We are having similar issue with bleaching strong AF488 signal esp. from structural markers. We've tried with LED light as well and hasn't helped much. Any suggestion would be highly appreciated. 
Best,
Presha

**Comment by radtkea** [comment, marked as answer]
Hi githubannimal and Presha!

Very sorry to get back to you so late. Thanks very much for asking your questions in the discussion forum.

Please see Troubleshooting table, incomplete fluorophore inactivation for tips/tricks: https://www.nature.com/articles/s41596-021-00644-9/tables/2

Since you are able to eliminate signal from AF555 and AF647, this tells me that you are 1) carefully removing all the mounting media from your samples, 2) preparing the LiBH4 correctly, and 3) successfully performing dye inactivation.

Presha is right! Very bright signals from structural markers and/or amplification with secondary antibodies can linger, unfortunately. 

I recommend the following: 
1. Titrate your antibody to reduce the signal to be in range with the other AF488 conjugates in your panel.
2. Perform several exchanges of 1 mg/ml LiBH4 (we have never increased the amount). Do 15 minutes of 1 mg/ml LiBH4, remove and use another aliquot prepared from the same vial, wait 15 minutes. 2 Exchanges should probably be sufficient.

LED light bleaching: The AF488 signal should disappear with LiBH4 alone. It's interesting that you see no reduction with LiBH4 and light! Try titrating your antibody and doing a few exchanges Presha. Another consideration, double check that there's no cross-reactivity between your antibodies. 

Lastly, if that doesn't work (and it should!), then you can always remove residual signal computationally by subtracting that channel from others. We use these free channel arithmetics: https://niaid.github.io/imaris_extensions/XTChannelArithmetic.html created by Ziv!

Best of luck and keep us posted! Thanks for asking in public on the discussion forum. You are helping others:)
A

**Reply by preshar** [reply]
Thanks Andrea for the detailed response/recommendations... we'll give it a try! Appreciate your help!

And thanks for creating the community knowledge-base. This is awesome! :)

Best,
Presha

**Reply by KoenigJFE** [reply]
Piggybacking on Andrea's comment, I wouldn't increase the concentration or amount of LiBH4 you're using. It is quite reactive and can result in an open flame if you don't follow the IBEX protocol carefully.

**Reply by radtkea** [reply]
Thanks Presha for your kind words! Glad you like it and it useful for you.

Thanks Josh! Yes, always follow the directions carefully. We have not experienced what you have described but we want to urge everyone to be safe and healthy! This is Dr. Koenig: https://www.youtube.com/watch?v=QgTBE7lfsbE
Very exciting and inspiring:)
